api/Document.json

```diff
@@ -1200,7 +1200,7 @@
             ],
             "opera": [
               {
-                "version_added": true
+                "version_added": "≤12.1"
```

Comment:
This is the only Presto change in this PR. Accidentally included?

Reply:
This is the only Presto change for this API from the collector, I figured it made sense to include it in this PR.  I've updated the title and description to assist.